### PR TITLE
Blog entry on developer workspace

### PR DESCRIPTION
**Pull request by FGS-CThomas**
Culture is a thing too so I thought I'd put something up on our workspace.

### CODE REVIEW

File: _posts/2016-04-08-Workspace.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+author: Christopher Thomas
+excerpt: At the end of last year we moved into a new workspace; it's pretty great.
+
+---
+
+![Cover]({{ site.baseurl }}/images/04-08-2016-Workspace/desk.jpg "Home sweet home") 
+
+Over the course of the last year or so the FoxGuard Software Engineering group has grown from a handful of employees scattered across myriad departments into a cohesive team with a tight-knit culture and an articulated ethos of professionalism and sustainability.  In the last quarter of 2015 the team moved from desks and offices strewn about one of our buildings to a purpose built development pit, bringing everyone together in one room and, in most cases, at one desk.
+
+In the short time since then we've seen a lot of real and tangible benefits.  Paired programming has become a common occurrence on the team and we've made a point to use it, not just to improve code quality, but to reduce "truck" or "lottery" factors – spreading knowledge around the team to eliminate a single point of failure.
```
Comment (fgs-dbudwin) [inline _posts/2016-04-08-Workspace.md]: May be beneficial to explain the "truck" or "lottery" factors.

File: _posts/2016-04-08-Workspace.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+In the short time since then we've seen a lot of real and tangible benefits.  Paired programming has become a common occurrence on the team and we've made a point to use it, not just to improve code quality, but to reduce "truck" or "lottery" factors – spreading knowledge around the team to eliminate a single point of failure.  
+
+The shared workspace has also improved team involvement in planning and architectural decisions. The perimeter of the work area is lined with white-boards which serve as a planning space but, adjacent as they are to everyone's desks, developers are able to enter and exit the conversation at will.  Headphones and chest-high walls prevent these conversations from derailing people who are in deep concentration while allowing engagement and participation on a case by case basis.
```
Comment (fgs-dbudwin) [inline _posts/2016-04-08-Workspace.md]: I would suggest hyphenation for "case-by-case"

File: _posts/2016-04-08-Workspace.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+In the short time since then we've seen a lot of real and tangible benefits.  Paired programming has become a common occurrence on the team and we've made a point to use it, not just to improve code quality, but to reduce "truck" or "lottery" factors – spreading knowledge around the team to eliminate a single point of failure.  
+
+The shared workspace has also improved team involvement in planning and architectural decisions. The perimeter of the work area is lined with white-boards which serve as a planning space but, adjacent as they are to everyone's desks, developers are able to enter and exit the conversation at will.  Headphones and chest-high walls prevent these conversations from derailing people who are in deep concentration while allowing engagement and participation on a case by case basis. 
+
```
Comment (fgs-dbudwin) [inline _posts/2016-04-08-Workspace.md]: "we've *built* a virtual environment"

File: _posts/2016-04-08-Workspace.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+title: 
+image: /images/04-08-2016-Workspace/desk.jpg
+author: Christopher Thomas
+excerpt: At the end of last year we moved into a new workspace; it's pretty great.
+
+---
+
+![Cover]({{ site.baseurl }}/images/04-08-2016-Workspace/desk.jpg "Home sweet home") 
+
+Over the course of the last year or so the FoxGuard Software Engineering group has grown from a handful of employees scattered across myriad departments into a cohesive team with a tight-knit culture and an articulated ethos of professionalism and sustainability.  In the last quarter of 2015 the team moved from desks and offices strewn about one of our buildings to a purpose built development pit, bringing everyone together in one room and, in most cases, at one desk.
```
Comment (FGS-BMarkham) [inline _posts/2016-04-08-Workspace.md]: I think "purpose built" should be hyphenated.

File: _posts/2016-04-08-Workspace.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+That is not to say that localization is the holy grail of the development process.  As FoxGuard has centralized our physical development workspace we have decentralized our virtual workspace.  By migrating our code repositories to GitHub and coordinating development activities through Slack, we've build a virtual environment in which developers can work from home when weather, traffic, or other commitments make getting to the office impossible.  
+
+There are still challenges to go: our Continuous Integration server lives inside the corporate network and occasionally needs someone to poke it, and our QA folks, while now physically integrated with the development team, are still only capable of working from home by remoting into their desktop machines.  Still, as summer approaches we have plenty of space, a flexible environment, and patio out back for... uh... meetings.  
+
+We wouldn't want to get ice cream on the carpets.
```
Comment (FGS-BMarkham) [inline _posts/2016-04-08-Workspace.md]: I would add a qualifier:

> We wouldn't want to get Friday ice cream on the carpets.

File: _posts/2016-04-08-Workspace.md (hunk 1 above)
Comment (FGS-CThomas) [inline _posts/2016-04-08-Workspace.md]: :white_check_mark:

File: _posts/2016-04-08-Workspace.md (hunk 2 above)
Comment (FGS-CThomas) [inline _posts/2016-04-08-Workspace.md]: :white_check_mark:

**Comment by FGS-CThomas** [comment]
Ready for re-review

**Comment by FGS-CThomas** [comment]
image

**Pull request merged**